sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuJsonTuple.scala

```diff
@@ -53,12 +54,17 @@ case class GpuJsonTuple(children: Seq[Expression]) extends GpuGenerator
   def generate(inputBatch: ColumnarBatch,
       generatorOffset: Int,
       outer: Boolean): ColumnarBatch = {
+    
+    val json = inputBatch.column(generatorOffset).asInstanceOf[GpuColumnVector].getBase.incRefCount
```

Comment:
Why are we incrementing the refcount here and doing a `withResource` on it down below?  Seems like we could simply not increment the refcount at all.  If for some reason we do need to do this, the increfcount needs to be as close as possible to the withResource to avoid leaks if an exception is thrown in-between these two operations.